ci/Jenkinsfile

```diff
@@ -35,9 +33,14 @@ pipeline {
       tagFilter: '*',
       type: 'PT_BRANCH'
     )
+    booleanParam(
+      name: 'FIND_BUILD',
```

Comment:
As I can see `FIND_BUILD` is not used - let's remove it,

Reply:
deleted, we will search build if the BUILD field is empty